eixampledb-core/src/main/java/com/eixampledb/core/EixampleDbBuilder.java

```diff
@@ -20,7 +20,18 @@ public EixampleDbBuilder withBacked(EixampleDbBackend backend) {
     }
 
     public EixampleDbBuilder withMapBackend() {
-        return withBacked(new EixampleDbMapImplementation());
+        EixampleDbMapImplementation e = new EixampleDbMapImplementation();
+        // Loading snapshot
+        try {
+            e.loadSnapshot("log.json");
+        } catch(Exception e1) {
```

Comment:
catch Exception is to wide, prefer to catch specific exception classes, otherwise we may silence bugs or errors.